test/unit/policy_test.rb

```diff
@@ -24,4 +24,34 @@ class PolicyTest < ActiveSupport::TestCase
     different_provider_policy = FactoryBot.build(:policy, name: persisted_policy.name, version: persisted_policy.version)
     assert different_provider_policy.save
   end
+
+  test 'validates that account is a tenant' do
```

Comment:
I would separate these assertions for better readability.
```
test 'simple buyer account is not a tenant' 
test 'master account is not a tenant' 
test 'simple provider account is a tenant' 
```

Reply:
I think this one makes sense to keep. It's 3 or 4 different ways to test the same small piece of code.